CleverPush/Source/CPBannerCardContainer.m

```diff
@@ -95,6 +95,9 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         CPAppBannerButtonBlock *block = (CPAppBannerButtonBlock*)self.blocks[indexPath.row];
 
         [cell.btnCPBanner setTitle:block.text forState:UIControlStateNormal];
+        if ([CPAppBannerModuleInstance getAppBannerVoucherCode] != nil) {
+            [cell.btnCPBanner setTitle:[CPAppBannerModuleInstance getAppBannerVoucherCode] forState:UIControlStateNormal];
```

Comment:
Don't just override the text. Please replace the text `{voucherCode}` with the voucherCodePlaceholder. The title could also be `This is your Voucher Code: {voucherCode}`. This can be done in a utility method.